SimCalorimetry/Configuration/python/SimCalorimetry_EventContent_cff.py

```diff
@@ -9,7 +9,9 @@
         'keep *_simHcalDigis_*_*', 
         'keep ZDCDataFramesSorted_simHcalUnsuppressedDigis_*_*',
         'drop ZDCDataFramesSorted_mix_simHcalUnsuppressedDigis*_*',
-        'keep *_simHcalTriggerPrimitiveDigis_*_*')
+        'keep *_simHcalTriggerPrimitiveDigis_*_*',
+        'keep *_mix_HcalSamples_*',
```

Comment:
They are 0 bytes for relvals because they only get produced if flags in HcalDigitizer are enabled